templates/partials/summary/summary.html

```diff
@@ -9,7 +9,7 @@
       {
         "groups": [
           {
-            "groupTitle": '<div id="' + group.id +'">' + group.title + '</div>' if group.title else None,
+            "groupTitle": '<div id="' + group.id +'">' + group.title + '</div>' if group.title else '',
```

Comment:
This is already working expected?. I can't make it display None when there isn't a title.